plugins/jobtype/src/azkaban/jobtype/HadoopSparkJob.java

```diff
@@ -478,24 +483,17 @@ private static void handleStandardArgument(Props jobProps,
     if (jobSparkVer != null) {
```

Comment:
I have added detail explanation of all possible scenarios in javadoc comment of getSparkLibConf method.